HomeTask20/HomeTask20/BasePage.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace HomeTask20
+{
+    public class BasePage
+    {
+        protected IWebDriver Driver;
+        public BasePage(IWebDriver driver)
+        {
+            this.Driver = driver;
+        }
+        public IWebElement FindElemtntByMethod(string path)
+        {
+            IWebElement element = null;
+            try
+            {
+                Waiting.WaitTheElement(this.Driver, path);
+                element = Driver.FindElement(By.XPath(path));
```

Comment:
I would recommend you to pass here parameter with By type instead of string. It will make you method more usable in case locators of web elements will have different  values (xpath, css, id etc.)

Reply:
Done